python/lib/scheduler/dmod/scheduler/scheduler.py

```diff
@@ -399,9 +411,10 @@ def _generate_docker_cmd_args(self, job: 'Job', worker_index: int) -> List[str]:
 
         # Finally, convert the args map to a list, with each "flag"/key immediately preceding its value
         args_as_list = []
-        for flag_key, param_val in docker_cmd_arg_map.values():
+        for flag_key, param_val in docker_cmd_arg_map.items():
```

Comment:
No clue how this ever worked - this would be trying to split the items in `[str(len(job.allocations)), self.build_host_list(job), str(job.job_id), str(worker_index)]` in two. Going with `items` yields the keys and values described in the comment.

Reply:
Yeah, great catch! This was for sure a bug.